be/src/vec/exprs/vliteral.cpp

```diff
@@ -118,6 +120,13 @@ void VLiteral::init(const TExprNode& node) {
             field = node.string_literal.value;
             break;
         }
+        case TYPE_JSON: {
+            DCHECK_EQ(node.node_type, TExprNodeType::JSON_LITERAL);
+            DCHECK(node.__isset.json_literal);
+            JsonValue value(node.json_literal.value);
```

Comment:
JsonValue is only used here, is it worth to create a class? maybe include json parser in JsonField is better.

Reply:
I do considered sole class `JsonField` is enough here,  but I think it's better to keep `JsonValue` and `JsonField` two different concepts, for that 
- `to_string` functions is needed in `JsonValue` and used in `VLiteral::debug_string()`
- `JsonField` is in memory representation of `JsonValue`, they have different semantics in logic
-  Consistence with implementation such as `Decimal` is good, which seems better for me